bip-MarcoPon-BlockchainURI.mediawiki

```diff
@@ -0,0 +1,80 @@
+<pre>
+  BIP: bip-MarcoPon-BlockchainURI
+  Title: URI scheme for Blockchain references / exploration
+  Author: Marco Pontello <marcopon@gmail.com>
+  Status: Draft
+  Type: Standards Track
+  Created: 29 August 2015
+</pre>
+
+==Abstract==
+
+This BIP propose an URI scheme for looking up blocks, transactions, addresses on a Blockchain explorer, or in general to make proper Blockchain references
+
+==Motivation==
+
+The purpose of this URI scheme is to enable users to handle all the requests for details about blocks, transactions, etc. with their preferred tool (being that a web service or a local application).
+Currently a Bitcoin client usually points to an arbitrary blockchain explorer when the user looks for the details of a transaction (es. Bitcoin Wallet uses BitEasy, Mycelium or Electrum uses Blockchain.info, etc.).
+Other times resorting to cut&paste is needed.
+The same happens with posts and messages that reference some particular txs or blocks, if they provide links at all.
+
+==Specification==
+
+The URI follow this simple form:
+
+ <nowiki>blockchain:[//chain]</type/hash></nowiki>
```

Comment:
ACK on defining the chain ID as "the first block hash that uniquely identifies the chain, usually the hash of the genesis block, but it may be the first incompatible block for forked chains" or something of the sort. I just think it would be nice to be able to link to this BIP when someone asks "what do you mean the chain ID?".
I'm fine with @MarcoPon's wording too, I'm not sure what's @btcdrak's concern with defining the chain ID (if any).